cli/src/utils/process.ts

```diff
@@ -19,8 +18,10 @@ export async function killProcess(processName: string) {
 }
 
 export async function killProcessByPort(port: number) {
-  const portArg = shellQuote.quote([`-i:${port}`]);
-  await execute(`lsof -t ${portArg} | while read line; do kill -9 $line; done`);
+  // NOTE(vadorovsky): The lint error in this case doesn't make sense. `port`
+  // is a harmless number.
```

Comment:
Looks like the only thing that could go wrong here is a negative value for the port number.

Reply:
True, let me add a check for that

Reply:
Done